charts/nucliadb_ingest/values.yaml

```
@@ -25,7 +25,7 @@ resources: {}
 serving:
   grpc: 8030
   monitor: 50101
-  metrics: 8081
+  metricsPort: 8081
 
```

We are missing here the new default value for serviceMonitor

```suggestion
serviceMonitor:
    enabled: false
```